protocol/directory.go

```diff
@@ -15,10 +15,15 @@ type ConiksDirectory struct {
 	policies merkletree.Policies
 }
 
+type UpdatePolicies struct {
+	EpDeadline merkletree.TimeStamp
```

Comment:
I'm a bit confused as to what `EpDeadline` represents now. I thought this is an actual time stamp for when clients should expect the next epoch, but I don't see it being updated every epoch (and I'm embarrassed I didn't notice before).